R/description.R

```diff
@@ -9,6 +9,9 @@
 #'   from `getOption("devtools.desc")`.
 #' @export
 use_description <- function(fields = NULL) {
+  if (file.exists(file.path(proj_get(), "DESCRIPTION"))) {
+    check_is_package("use_description()")
+  }
```

Comment:
Necessary because this function is used to create DESCRIPTION in the first place.